test/evergreen.yml

```diff
@@ -968,6 +968,21 @@ tasks:
 
             ${test_env_vars|} $(pwd)/test/cppsuite/run -t hs_cleanup -C 'debug_mode=(cursor_copy=true)' -f test/cppsuite/configs/hs_cleanup_default.txt -l 2
 
+  - name: cppsuite-search-near01-default
```

Comment:
Can you follow the name of the file, i.e `cppsuite-search-near-01-default`? Unless it's a bad idea. I don't have a strong opinion on this. 
You also need to add it to the list [here](https://github.com/wiredtiger/wiredtiger/blob/develop/test/evergreen.yml#L2861).